### PR TITLE
[API] PrefabHelper

### PR DESCRIPTION
**Pull request by Misfiy**

### CODE REVIEW

**Comment by VALERA771** [comment]
I've put requires testing cuz not sure that Network client will work on server :)

**Review by BoltonDev — APPROVED** [review]

**Review by VALERA771 — APPROVED** [review]

**Comment by VALERA771** [comment]
Maybe add spawn to game entity? Cuz I think all of them (or at least a lot) supports spawning via Network server

**Comment by BoltonDev** [comment]
> Maybe add spawn to game entity? Cuz I think all of them (or at least a lot) supports spawning via Network server

This is not needed since there is already a method in PrefabHelper

**Comment by VALERA771** [comment]
> > Maybe add spawn to game entity? Cuz I think all of them (or at least a lot) supports spawning via Network server
> 
> 
> 
> This is not needed since there is already a method in PrefabHelper

But then method in controller should be removed as I think or just edits so it'll use that method

**Review by VALERA771 — APPROVED** [review]
Seems good but why we have spawn methods in workstation and generator?

**Comment by Misfiy** [comment]
> Seems good but why we have spawn methods in workstation and generator?
Multiple others did, and I thought it would be useful to also easily getting a workstation + more easy to find

File: Exiled.API/Features/Generator.cs [hunk 1]
```diff
@@ -216,6 +226,18 @@ public KeycardPermissions KeycardPermissions
             set => Base._requiredPermission = (Interactables.Interobjects.DoorUtils.KeycardPermissions)value;
         }
 
+        /// <summary>
+        /// Spawns a <see cref="Generator"/>.
+        /// </summary>
+        /// <param name="position">The position to spawn it at.</param>
+        /// <param name="rotation">The rotation to spawn it as.</param>
+        /// <returns>The Generator that was spawned.</returns>
```
Comment (Vladislav-CS) [inline Exiled.API/Features/Generator.cs:234]:
```suggestion
        /// <returns>The <see cref="Generator"/> that was spawned.</returns>
```

**Review by Vladislav-CS — CHANGES_REQUESTED** [review]

**Review by Vladislav-CS — APPROVED** [review]

**Pull request merged**